cpp/ps0003/ps0003_sol.cpp

```diff
@@ -4,14 +4,17 @@ using namespace std;
 
 int main()
 {
-    short s ;
-    cin >> s ;
-     if ( s % 2 == 0 ) {
+    // prompting user for number 2/2
+    short s;
+    cin >> s;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number